### PR TITLE
Upgrade for symfony 3

### PR DESCRIPTION
**Pull request by lepiaf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
         "psr-4": { "ETS\\DocumentStorageBundle\\": "" }
     },
     "require": {
-        "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.2",
-        "ets/document-storage": "~0.5"
+        "php": ">=5.5.9",
```
Comment (ch3ric) [inline composer.json]: nope

File: composer.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
...
+        "symfony/framework-bundle": "~3.1",
```
Comment (ch3ric) [inline composer.json]: `>=2.2`

File: composer.json [hunk 3]
```diff
@@ -14,8 +14,8 @@
...
+        "ets/document-storage": "~0.6"
```
Comment (ch3ric) [inline composer.json]: nope

**Comment by ch3ric** [comment]
create tag 0.5.1

File: composer.json (hunk 1 above)
Comment (lepiaf) [reply composer.json]: done

**Pull request merged**